RELEASING.md

```diff
@@ -4,8 +4,11 @@ The PHP library and extension are released independently of each other.
 
 ## Packagist Package
 
+1. Bump the `VERSION` constant in [`StackdriverExporter`][exporter]
```

Comment:
Doesn't this need to be kept in sync with something that defines the actual version in packagist?

Reply:
Packagist goes off of repository tags (which should be synced to a corresponding GitHub release)

Reply:
ah okay.